flash/core/data/utilities/loading.py

```diff
@@ -73,7 +73,13 @@ def _load_image_from_image(file):
 
 
 def _load_image_from_numpy(file):
-    return Image.fromarray(np.load(file).astype("uint8")).convert("RGB")
+    arr = np.load(file)
+    if not (arr == arr.astype("uint8")).all():
```

Comment:
This is not very controlled/consistent, why only uint8 shall be scaled to 0-255 and not float from 0-1?